src/chrome/content/rules/idleleo.com.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="idleleo">
+	<target host="d.idleleo.com" />
```

Comment:
https://travis-ci.org/EFForg/https-everywhere/jobs/586022926#L325

Move to comments.